### ISSUE TITLE
publish images to dockerhub or some other registry

### ISSUE THREAD

**Issue by geekflyer** [VividCortex/docker#16]
It's bad onboarding experience when one first has to build and publish the docker image.

**Comment by jomendoz** [comment]
I agree. Datadog agents are provided in Docker Hub. And I considered VC agent on Docker to be a faster option rather than setting up an EC2 instance, but I think I wouldn't like to maintain a VC image in our own registries.
